tsdb/db.go

```diff
@@ -588,9 +588,9 @@ func (db *DB) run() {
 			return
 		case <-time.After(backoff):
 		}
-
+		timer := time.NewTimer(1 * time.Minute)
```

Comment:
Ticker makes more sense here. A timer would run the compaction only once.

Reply:
Considering backoff logic it isn't that easy to convert per-iteration timer to one ticker.

Reply:
Ah sorry, I didn't notice the loop enclosed this. In that case, the current implementation looks simpler and `time.After` uses `time.NewTimer` internally. 1 object per minute doesn't hurt much.